mac/Makefile

```diff
@@ -13,12 +15,17 @@ libwooting-rgb-sdk.dylib: $(OBJS)
 	$(CC) $(LDFLAGS) $(LIBS) -shared -fPIC -Wl,-install_name,$0.0 $^ -o $@
 
 $(OBJS): %.o: %.c
-	$(CC) $(CFLAGS) -c $(INCLUDES) $< -o $@
+	$(CC) $(CPPFLAGS) $(CFLAGS) -c $(INCLUDES) $< -o $@
 
 clean:
 	rm -f $(OBJS)
 
 install: libwooting-rgb-sdk.dylib
-	cp libwooting-rgb-sdk.dylib /usr/local/lib/libwooting-rgb-sdk.dylib
+	mkdir -p $(prefix)/lib
+	cp libwooting-rgb-sdk.dylib $(prefix)/lib/libwooting-rgb-sdk.dylib
+	chmod 755 $(prefix)/lib/libwooting-rgb-sdk.dylib
```

Comment:
Noob question, can you explain why the write permissions for the user?

Reply:
Right, it shouldn't make a difference in practice, because root can write anyway, but I've followed the permissions of the rest of the libraries on my Linux in `/usr/lib`. (Strictly speaking, root also needs to "write" that file on `make install`)

The reason why I set the permissions at all, is that the user might have a more restrictive umask, and cp also copies the file's permissions, so it might be that the installed library will be `root:root rwx------` (with umask `0077`), making the library unusable for regular users.